README.md

```diff
@@ -69,6 +69,32 @@ The module will define a cron job that runs daily to assure _appointment\_horizo
 
 As of release 0.6.0, the most precise way to make appointment blocks is to set _appointment\_horizon_ days to zero and set _start\_date_ to the precise date for which you want to make appointments. Select the sites that need new appointment blocks and click `Manually generate new appointments`. Repeat this process for each site date that needs new appointment blocks.
 
+## Showing Appointment Availability
+
+A table showing the current number of available appointments can be shown on any survey page after a few steps. Some SQL knowledge is required.  
+![Appointment availabilty table example](images/appointment_table.png)
+1. Create a query with [REDCap Webservices](https://github.com/ctsit/redcap_webservices) to deliver the data you wish to display.  
+    example:
+    ```sql
+    CONCAT(site_long_name, ' (', site_short_name, ')') AS location,
+        ts.testing_type,
+        COUNT(IF(fra.record_id IS NULL, 1, NULL)) AS available
+            FROM redcap_entity_fr_appointment AS fra
+            INNER join redcap_entity_test_site AS ts ON (fra.site = ts.id)
+            WHERE ts.project_id = [project-id]
+            AND ( fra.appointment_block > UNIX_TIMESTAMP( DATE( NOW() + INTERVAL IF(HOUR(NOW()) >= 16, 2, 1) DAY ) ) )
+            GROUP BY site
+    ```
```

Comment:
A little extra white space improves the formatting in MacDown and should not impede to the formatting in Github.

```suggestion
    ```

```